source/request-as-event-emitter.ts

```diff
@@ -1,3 +1,4 @@
+import {ReadStream} from 'fs';
```

Comment:
Generally, it's nicer to rename generic imports like this as it's not immediately clear further down in the code that it's an **fs** read stream.

```
import {ReadStream: FsReadStream} from 'fs';
```

Reply:
Will do :)